<launch>

	<param name="pc_name" type="string" value="YOUR_POINTCLOUD.pcd" />
	<rosparam command="load" file="$(find ImMesh)/config/offline_pointcloud.yaml" />
	<param name="meshing/distance_scale" type="double" value="1.0" /> 
	<param name="gui_font_size" type="int" value="20" /> 
	
	<node pkg="ImMesh" type="ImMesh_mapping" name="laserMapping" output="screen" launch-prefix=""> 
	</node>

</launch> 	
